app/macos/Runner/AppDelegate.swift

```diff
@@ -214,6 +208,29 @@ class AppDelegate: FlutterAppDelegate {
         }
     }
     
+    private func saveDestinationFolderAccess(_ folderPath: String) throws {
+        let folderURL = URL(fileURLWithPath: folderPath)
+        let bookmarkData = try folderURL.bookmarkData(
+            options: [.withSecurityScope],
+            includingResourceValuesForKeys: nil,
+            relativeTo: nil
+        )
+        Defaults[.destinationFolderBookmark] = bookmarkData
```

Comment:
Since we store the bookmark, can we revoke the access to the old folder before granting a new one?

Reply:
See now